aiohttp/connector.py

```diff
@@ -885,6 +893,8 @@ async def _resolve_host(
         resolved_host_task = asyncio.create_task(
```

Comment:
Unrelated but if forgot about this one. It could also be an eager task.  I likely didn’t see it in the profile anymore because I did a PR a while back to not create a task if it’s cached and will be most of the time